app/views/evaluator_submission_assignments/_submission_row.html.erb

```diff
@@ -0,0 +1,19 @@
+<tr data-evaluator-id="<%= evaluator.id %>" data-evaluator-type="user">
+  <td data-label="Submission ID" class="text-top text-bold"><%= assignment.submission.id %></td>
+  <td data-label="Evaluation status">
+    <span class="text-top text-bold <%= evaluation_status(assignment.status) %>"><%= assignment.status.titleize %></span>
```

Comment:
did you try using the [uswds tag](https://designsystem.digital.gov/components/tag/) here (with color)?